web/src/components/profile/ProfileNaicsCode.test.tsx

```diff
@@ -52,18 +55,33 @@ describe("<ProfileNaicsCode />", () => {
     expect(screen.getByText("123456")).toBeInTheDocument();
   });
 
-  it("Display Edit when there is naics code", () => {
+  it("display Edit when there is naics code", () => {
     renderComponent({ naicsCode: "624410" });
     expect(screen.getByText(configForField.editText)).toBeInTheDocument();
     expect(screen.queryByText(configForField.addText)).not.toBeInTheDocument();
   });
 
+  it("doesn't display Edit text and displays tooltip when there is a naics code and operating phase is up and running or up and running owning", () => {
```

Comment:
Is this correct?

If a user is a Poppy who has graduated to UP_AND_RUNNING and has not yet done their tax access lookup, shouldn't this still be editable?